### PR TITLE
{math} [intel/2018a] PyStan v2.19.0.0 for Python/3.6.4 {REVIEW}

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/p/PyStan/PyStan-2.19.0.0-intel-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+dependencies = [
+    ('Python', '3.6.4'),
+]
+
+download_dep_fail = True
+
+use_pip = True
+
+options = {'modulename': '%(namelower)s'}
```
Comment (smoors) [inline easybuild/easyconfigs/p/PyStan/PyStan-2.19.0.0-intel-2018a-Python-3.6.4.eb]: @SimonPinches the default modulename is already the lowercase name, so you can remove this line.

**Review by smoors — DISMISSED** [review]

File: easybuild/easyconfigs/p/PyStan/PyStan-2.19.0.0-intel-2018a-Python-3.6.4.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/p/PyStan/PyStan-2.19.0.0-intel-2018a-Python-3.6.4.eb]: Thanks @smoors, I just (force) pushed an update to correct this.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2627.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c6e8933860e360e986b08e4a2744ade3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3250.victini.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/aeb7ff3737c7ca0b41039581464b03cb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2033.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7a0108c709582c9591d2cbe52e11372c for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**